web/html/src/manager/products.js

```diff
@@ -423,7 +423,7 @@ class Products extends React.Component {
   };
 
   searchData = (datum, criteria) => {
-    if (criteria) {
+    if (criteria && datum.label) {
```

Comment:
Is this change related to this PR? It doesn't look like. If so, please remove it from here and add the fix (that makes sense) into a separate commit and PR. Thank you.

Reply:
Sure, I will do it :)